runtime/wasm/src/host_exports.rs

```diff
@@ -11,35 +11,14 @@ use graph::prelude::serde_json;
 use graph::prelude::{slog::b, slog::record_static, *};
 use semver::Version;
 use std::collections::HashMap;
-use std::fmt;
 use std::ops::Deref;
 use std::str::FromStr;
 use std::time::{Duration, Instant};
 use web3::types::H160;
 
 use graph_graphql::prelude::validate_entity;
 
-use crate::module::WasmiModule;
-
-pub(crate) trait ExportError: fmt::Debug + fmt::Display + Send + Sync + 'static {}
-
-impl<E> ExportError for E where E: fmt::Debug + fmt::Display + Send + Sync + 'static {}
-
-/// Error raised in host functions.
-#[derive(Debug)]
-pub(crate) struct HostExportError<E>(pub(crate) E);
```

Comment:
This was necessary because we needed a local type to implement the wasmi error trait for. Wasmtime doesn't require any of that, so this became unnecessary. A specialized error type may be useful for us in the runtime (deterministic vs non-deterministic errors come to mind), but this wasn't it.